docs/assets/mainnet.json

```diff
@@ -591,6 +591,15 @@
     "locale": "in",
     "port": "10332",
     "type": "RPC"
+   },
+   {
+    "protocol": "http",
+    "url": "seed01.cryptomunki.tech",
+    "location": "Hong Kong",
+    "address": "40.83.123.178",
+    "locale": "hk",
```

Comment:
I think it would be better to use "cn" as locale here toi show China flag (Unless your goal here was to add HK flag)